### ISSUE TITLE
Tests for twisted.pair should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7722]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7722|
|Type|enhancement|
|Created|2014-11-26 10:29:12Z|
|Branch|https://github.com/twisted/twisted/tree/pair-tests-names-7722|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7722.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a36/a361fd27bb8acb7581c07327c33327c845bfa7fb/c67ca9818fc3f6e96bfb13126e975ddd1473f8c0.patch) (2479 bytes) - added by eeshangarg on 2014-11-26 10:32:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7722 7722
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_pair_tests_names_7722 branches/pair-tests-names-7722
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__pair pair
keywords__easy easy
time__1416997752167171 1416997752167171
changetime__1417034069885703 1417034069885703
version__None None
owner__exarkun exarkun
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7722.patch​) that changes all of the tests in twisted.pair to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [43512]) Branching to 'pair-tests-names-7722'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [43513]) Apply bugfix_7722.patch 

Refs [#7722](https://github.com/twisted/twisted/issues/7722)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thanks.  These changes look fine to me.  Buildbot looks pretty green.  I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [43515]) Merge pair-tests-names-7722 

Author: eeshangarg
Reviewer: exarkun
Fixes: [#7722](https://github.com/twisted/twisted/issues/7722)

Rename all of the test case classes in Twisted Pair to conform to the
recently documented test case class naming convention.
